ambry-frontend/src/main/java/com.github.ambry.frontend/AmbryBlobStorageService.java

```diff
@@ -67,6 +76,10 @@
   private static final String OPERATION_TYPE_HEAD = "HEAD";
   private static final String OPERATION_TYPE_DELETE = "DELETE";
   private static final String OPERATION_TYPE_POST = "POST";
+  private static final Set<String> requiredAmbryHeadersForOldPuRequest =
```

Comment:
Renamed to `requiredAmbryHeadersForPutWithServiceId`. The other one is renamed to `requiredAmbryHeadersForPutWithAccountAndContainerName`